### ISSUE TITLE
Empty cell is not generated????

### ISSUE THREAD

**Issue by xiaofengzang** [chjj/marked#476]
I have following mark down file which has a table, notice the 3rd column does not have value, and is empty

| Header1 | Header2 | Header3 | Header4 |
| :-- | :-: | :-- | :-- |
| value1 | value2 |  | value4 |

it generates following html, notice it has 3 cells, I would expect there is an empty <td></td> after value2.  Anyone know what could be issue here?

<table>
..............
<tbody>
<tr>
<td align="left">value1</td>
<td align="center">value2</td>
<td align="left">value4</td>
</tr>
</tbody>
</table>

**Comment by suan** [comment]
+1

This issue is preventing me from using marked for [instant-markdown-d](https://github.com/suan/instant-markdown-d)

**Comment by joshbruce** [comment]
#980

**Comment by Feder1co5oave** [comment]
Can't reproduce this without the markdown source you're feeding marked.
I tried something like this and it is parsed as expected:
```markdown
| h1 | h2 | h3 | h4 |
| -- | -- | -- | -- |
| c1 | c2 |    | c4 |
```

~~~html
<table>
<thead>
<tr>
<th>h1</th>
<th>h2</th>
<th>h3</th>
<th>h4</th>
</tr>
</thead>
<tbody>
<tr>
<td>c1</td>
<td>c2</td>
<td></td>
<td>c4</td>
</tr>
</tbody>
</table>
~~~

**Comment by kaelig** [comment]
Seems like this is still happening.

See https://marked.js.org/demo/?text=%7C%20a%20%7C%20b%20%7C%20c%20%7C%0A%7C%20-%20%7C%20-%20%7C%20-%20%7C%0A%7C%20a%20%7C%20%20%20%7C%20c%20%7C%0A

```md
| a | b | c |
| - | - | - |
| a |   | c |
```

Care to reopen?

**Comment by UziTech** [comment]
@kaelig this was fixed in #1262 which has not been released yet

**Comment by kaelig** [comment]
Oh great! Thank you!

**Comment by SilentGert** [comment]
Hi,
I'm running into the same issue. Is there already a timeframe when the patch will be released?

Thanks

**Comment by joshbruce** [comment]
@styfle: Are all the PRs we want in 0.5.0 in? I know we run on continuous delivery but we also tend to have a few more submitted before actually performing the release.

/cc @UziTech @davisjam

**Comment by styfle** [comment]
I'm happy now that we landed #1315

Ship it :shipit:
